### DISCUSSION TITLE
"Manual" servers

### DISCUSSION THREAD

**Discussion by Danielss89** [vitobotta/hetzner-k3s#288] [Q&A]
Hello

Is it possible to use "manual" servers?
I would like to have my main nodes in Hetzner Cloud, but i would also like to have nodes in hongkong, singapore and sydney, and i found another cloud provider which can give that.
So i would just create them manually at the cloud provider, but how would i add them to the cluster?
Anywhere in the scripts i can do this?

**Comment by vitobotta** [comment]
Hi, I am planning to add support for Hetzner dedicated servers since they can be attached to the same private network as cloud servers, but there is no plan at this stage to support adding nodes from other providers because several things would not work. Namely: load balancing, persistent volumes, autoscaling. It needs quite a bit of though to implement workarounds.
